Mage.Sets/src/mage/cards/f/FieldOfRuin.java

```diff
@@ -91,7 +91,7 @@ public boolean apply(Game game, Ability source) {
                 if (player != null) {
                     TargetCardInLibrary target = new TargetCardInLibrary(0, 1, StaticFilters.FILTER_CARD_BASIC_LAND);
                     if (player.searchLibrary(target, game)) {
-                        player.moveCards(new CardsImpl(target.getTargets()), Zone.BATTLEFIELD, source, game);
+                        player.moveCards(new CardsImpl(target.getTargets()).getCards(game), Zone.BATTLEFIELD, source, game);
```

Comment:
Original moveCards method calls .getCards(game) itself. So new code is equivalent to the old code, isn't it?

Reply:
My thought process was that the code between Field Of Ruin and Collective Voyage should be identical as they do the same functionality.  Collective voyage works, and Field was being reported as not functioning/bugged.  So even if this code does the same thing at least we’re consistent.